frontend/src/app/main/ui/workspace/tokens/components/controls/input_tokens_value.cljs

```diff
@@ -19,27 +20,34 @@
    [:placeholder {:optional true} :string]
    [:value {:optional true} [:maybe :string]]
    [:class {:optional true} :string]
+   [:color {:optional true} [:maybe :string]]
    [:error {:optional true} :boolean]])
```

Comment:
You have added a prop `on-display-colorpicker` but you didn't add it to the schema